apps/bttester/syscfg.yml

```diff
@@ -87,6 +87,7 @@ syscfg.defs:
         value: '"test_broadcast"'
 
 syscfg.vals:
+    BLE_AUDIO_MAX_CODEC_RECORDS: 5
```

Comment:
This won't be needed if you remove lc3 package

Reply:
Looking at my previous comment https://github.com/apache/mynewt-nimble/pull/1747#discussion_r1562444712 do we need to remove lc3 package?

Reply:
LC3 registers 2 codec entries by default. Does this need to be 5? If you only add one more codec, 3 would suffice

Reply:
Agree that 5 is too much, this will be removed in upcoming patch.

Reply:
Done